frontend/src/components/TwoSlotDisplay.js

```diff
@@ -0,0 +1,143 @@
+import React, { useReducer } from "react";
+import PropTypes from "prop-types";
+import EmptyRooms from "./EmptyRooms.js";
+import { Slots, DayNames, Complexes, Floors } from "../constants/constants.js";
+import { getNextSlot, getInitialChillPlaceDetails } from "../util/utilities.js";
+import { CHILLPLACE_DISPATCH_TYPES } from "../constants/constants.js";
+
+const chillPlaceDetailsReducer = (state, action) => {
```

Comment:
What is the need to use `chillPlace` in the function name? If needed we can use `chillzone` (name of repository). here even that is not necessary. Even constant type names do not need it.

Reply:
Okay, I named it as just `reducer` now. Does this work?